test/test_macho.rb

```diff
@@ -313,8 +321,15 @@ def test_change_install_name
 
       file.write(actual)
 
-      # compare actual and expected file hashes, to ensure file correctness
-      assert equal_sha1_hashes(actual, expected)
```

Comment:
Oh! That's what I get for only paying attention to the diff comment:

```diff
@@ -313,8 +321,15 @@ def test_change_install_name
```

I'll have this corrected tonight :smile: